src/sentry/api/event_search.py

```diff
@@ -1417,7 +1417,7 @@ def get_json_meta_type(field_alias, snuba_type, function=None):
     return snuba_json
 
 
-FUNCTION_PATTERN = re.compile(r"^(?P<function>[^\(]+)\((?P<columns>[^\)]*)\)$")
+FUNCTION_PATTERN = re.compile(r"^(?P<function>[^\(]+)\((?P<columns>.*)\)$")
```

Comment:
Arguments to `to_other` can include `)`, so only assert string ends with `)`, and anything in between is matched to columns.